Simplenote/SimplenoteSimpleTextPrintFormatter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SimplenoteSimpleTextPrintFormatter.swift
+//  Simplenote
+//
+//  Created by Charlie Scheer on 1/10/20.
+//  Copyright © 2020 Automattic. All rights reserved.
+//
+
+import UIKit
+
+class SimplenoteSimpleTextPrintFormatter: UISimpleTextPrintFormatter {
```

Comment:
Can we rename **SimplenoteSimpleTextPrintFormatter** > **SPSimpleTextPrintFormatter**  (OR) **SimpleTextPrintFormatter**?

ObjC convention is usually the app's initials + name, for namespaces purposes. Suppose a dependency used **SimpleTextPrintFormatter**, if you had a class with the exact same name, you'd get a compiler error.

In Swift that's not really needed anymore, each framework is namespaced by default (UIKit.UISimpleTextPrintFormatter)

That being said, since Simplenote is in a mixed state (ObjC / Swift), we're keeping around the SP prefix for a little longer (OR droppping it entirely, depending on how cool one looks, or the other, to be honest!).